src/app/components/team/TeamIntegrations.js

```diff
@@ -17,6 +17,22 @@ const TeamIntegrations = () => (<QueryRenderer
         current_team {
           id
           dbid
+          api_keys(first: 10000) {
```

Comment:
TODO: Convert to fragment

Reply:
Until here this is the only comment I would add to my review! :D